parsers/src/main/scala/uk/gov/ons/addressIndex/parsers/Tokens.scala

```diff
@@ -13,16 +13,26 @@ object Tokens extends CrfTokenable {
 
   override def normalise(tokens: Tokens): Tokens = tokens map (_.toUpperCase)
 
+  val OrganisationName: Token = "OrganisationName"
```

Comment:
OK, but it looks more like a `Label` to me (especially in the context above in the `CrfTokenResult`: ` label: Token`